content/basics/index.md

```diff
@@ -497,16 +454,15 @@ try {
    // an error occurred
    const message = e.message;
 } finally {
-   // do something that u would do either way
+   // do something that you would do either way
 }
 
 // can create own errors or throw errors like so.
 const e = new Error('Bad Thing Happened');
+throw e;
 
-// And there's also the 'throw' syntax
-// you don't even need to create a custom error you can simply throw signal
-// something went wrong.
-throw 'Error';
```

Comment:
I've never seen this been done in productionised code, so I don't think it needs to be taught.

Reply:
I'll remove!